applications/tuxedo.md

```diff
@@ -0,0 +1,185 @@
+  owner: Address,
+  /// The data associated with this output. In the simplest case, this will be a token balance, but could be arbitrarily rich state.
+  data: Vec<u8>,
+}
+
+/// A single input references the output to be consumed or peeked at and provides some witness data, possibly a signature.
+struct Input {
+  /// A previously created output that will be consumed by the transaction containing this input.
+  output: OutputId,
+  /// A witness proving that the output can be consumed by this input. In many cases including that of a basic cryptocurrency, this will be a digital signature.
+  redeemer: Vec<u8>,
+}
+```
+
+The core of the API exposed developers who create Tuxedo Pieces, will roughly follow this trait. We expect this will have to get more specific as our development shows us what we haven;t yet considered.
+
+```rust
+/// The API of a Tuxedo Piece
+trait TuxedoPiece {
+
+  /// The type of data stored in Outputs associated with this Piece
+  type Data;
+
+  /// The validation function to determine whether a given input can be consumed.
+  fn validate(transaction: Transaction, input: Input) -> bool;
+}
+```
+
+This grant does not strive to create the entire rich ecosystem of Tuxedo pieces that we hope to eventually be developed on top of Tuxedo. Rather it strives to create the core of the Tuxedo system and a few of the most important and exemplary pieces. Specifically, we strive to develop the analogs to FRAME Executive, FRAME System, Pallet Balances, and Pallet Transaction Payment.
+
+### Ecosystem Fit
+
+Tuxedo is a framework for writing Substrate runtimes. Substrate is the toolkit for building virtually all parachain nodes as well as many standalone blockchains. As such, Tuxedo provides a richer set of options to runtime developers, and hopes to attract teams to the Substrate / Polkadot ecosystem who may have otherwise gone elsewhere.
+
+The primary users of Tuxedo will be parachain and runtime developers who will use Tuxedo directly to structure their runtimes. Of course, the user base will trickle downstream as well to users of those parachains that choose to build with Tuxedo. However, chain users will use Tuxedo only indirectly.
+
+There are no projects like this in the Substrate ecosystem, although they do exist in the broader blockchain space; Cardano being the most notable example.
+
+While it fulfills a similar role, Tuxedo is not intended to compete with FRAME, but rather to compliment it, by welcoming projects that fit naturally with the utxo model into the Substrate ecosystem, as FRAME does for projects that fit the accounts model.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Joshy Orndorff https://github.com/JoshOrndorff
+- Andrew Burger https://github.com/coax1d
+
+### Contact
+
+- **Contact Name:** Joshy Orndorff
+- **Contact Email:** joshyorndorff at proton dot me
+- **Registered Address:** The address you'll use in the invoices
```

Comment:
You should actually put the address here ;)
```suggestion
- **Registered Address:** -
```

Reply:
I wondered if that's what you meant. I'm supposed to publish my physical location on the Earth for everyone to see? I'd prefer not to if that's at all possible. I see that the FuzzLand application put N/A. Is that acceptable?

Reply:
That's fine. We only need the address as part of the invoice (which is not public).